app/src/main/kotlin/cz/covid19cz/erouska/DI.kt

```diff
@@ -53,8 +56,11 @@ val viewModelModule = module {
 }
 
 val databaseModule = module {
-    fun provideDatabase(application: Application): AppDatabase {
+    fun provideDatabase(prefs: SharedPrefsRepository, application: Application): AppDatabase {
+        val key = getDatabaseEncryptionKey(prefs, application.baseContext)
```

Comment:
We also need migration from not-encrypted database to encrypted, since this will be an update.